mnzube/Banka
# #165345198 Reviews UI changes

## What does this PR do?
Reviews changes done on the UI
## Description of Task to be completed?
Redirects pages from the home page with correct links.
Adds content to pages.
## How should this be manually tested?
check links, admin page, password reset page
By clicking on the Welcome page to cashier, sign in heading to 
## Any background context you want to provide?
This PR adds content according to the project owner reviews.
## What are the relevant pivotal tracker stories?
[165345198](https://www.pivotaltracker.com/story/show/165345198)
## Screenshots (if appropriate)
